Atomic/sign.py

```diff
@@ -0,0 +1,107 @@
+from . import util
+from . import Atomic
+import os
+import tempfile
+from .atomic import AtomicError
+import re
+
+
+class Sign(Atomic):
+    def sign(self):
+        # TODO
+        # Atomic is run as sudo. Should we work around that?
```

Comment:
Yes I think the comment should say if atomic is run as root, nothing to do with sudo.  If run as root we could try to figure out the homedir of the user running the command.  SUDO_USER and /proc/self/login_uid, are good indicators of the users UID.